app/main/spawn.js

```diff
@@ -18,7 +18,7 @@ import BUILD_CONFIG from '../../build-config';
 const ROOT = path.join(__dirname, '..', '..');
 const DB_PATH = ROOT;
 const SERVICES_PATH = path.join(ROOT, 'lib', 'services');
-const UA_SERVICE_BIN = path.join(SERVICES_PATH, 'user-agent-service', 'bin', 'user-agent-service');
+const UA_SERVICE_BIN = path.join(SERVICES_PATH, 'user-agent-service', 'user-agent-service');
```

Comment:
That's the target file for webpack, I could still keep the bin directory around but it seems unnecessary once everything else is gone.